avocado/utils/cloudinit.py

```diff
@@ -132,12 +146,28 @@ def do_POST(self):
         self.send_response(200)
 
     def log_message(self, format_, *args):  # pylint: disable=W0221
+        """Logs an arbitrary message.
+
+        :note: It currently disables any message logging.
+        """
         pass
 
 
 class PhoneHomeServer(HTTPServer):
+    """Implements the phone home HTTP server.
+
+    Wait the phone home from a given instance.
+    """
 
     def __init__(self, address, instance_id):
+        """
```

Comment:
I suggest adding at least one summary line, otherwise it could break some documentation tools, like sphinx.

Reply:
ok, gonna send another version.